enterprise/cypher/acceptance-spec-suite/pom.xml

```diff
@@ -89,6 +89,27 @@
       <artifactId>scalacheck_2.11</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
when we run our tests we run them in parallel fashion and so far latest junit and latest surefire can't do it (forked VMs will die in any moment junit will corrupt their sdtout). How this is achieved here?